src/Element/HasPopoverTargetAction.php

```
@@ -0,0 +1,46 @@
+     * $element->popoverTargetAction($action);
+     * $element->popoverTargetAction(null);
+     * ```
+     *
+     * @param string|Stringable|UnitEnum|null $value Popover target action (`hide`, `show`, `toggle`), or `null` to
+     * remove the attribute.
+     *
+     * @return static New instance with the updated `popovertargetaction` attribute.
+     */
+    public function popoverTargetAction(string|Stringable|UnitEnum|null $value): static
+    {
+        Validator::oneOf($value, PopoverTargetAction::cases(), ElementAttribute::POPOVERTARGETACTION);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Helper\Validator' in method 'popoverTargetAction'. (undefined)

(StaticAccess)